frontend/src/features/admin-form/common/components/AdminFormNavbar/AdminFormNavbarContainer.tsx

```diff
@@ -52,8 +54,8 @@ const useAdminFormNavbar = () => {
   )
 
   const handlePreviewForm = useCallback((): void => {
-    console.log('preview form button clicked')
-  }, [])
+    navigate(`/${formId}/${ADMINFORM_PREVIEW_ROUTE}`)
+  }, [navigate, formId])
```

Comment:
I've kept it as `IconButton` in `AdminFormNavbar` because I thought it would be easier to standardise the formatting that way (same as the "Manage Collaborators" `IconButton` beside it). Hence have left it as `navigate`